packages/components/src/components/input-number/input-number.tsx

```diff
@@ -232,7 +232,7 @@ export class InputNumber
    * specifies the maximum length of text for the component's value.
    *
    * @mdn [maxlength](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#maxlength)
-   * @deprecated This property has no effect on the component.
+   * @deprecated in v3.0, removal target v6.0 - This property has no effect on the component.
```

Comment:
Consistency suggestion:
```suggestion
   * @deprecated in v3.0.0, removal target v6.0.0 - This property has no effect on the component.
```